src/libs/ReportUtils.ts

```diff
@@ -4761,6 +4762,23 @@ function canBeAutoReimbursed(report: OnyxEntry<Report>, policy: OnyxEntry<Policy
     return isAutoReimbursable;
 }
 
+/**
+ * Checks if report chat contains add bank account action
+ */
+function hasAddBankAccountAction(iouReportID: string): boolean {
+    const reportActions = ReportActionsUtils.getAllReportActions(iouReportID);
+    const isSubmitterOfUnsettledReport = isCurrentUserSubmitter(iouReportID) && !isSettled(iouReportID);
+    const hasCreditBankAccount = store.hasCreditBankAccount();
+
+    if (!isSubmitterOfUnsettledReport || hasCreditBankAccount) {
+        return false;
+    }
+
+    return !!Object.values(reportActions).find(
+        (action) => action.actionName === CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENTQUEUED && (action.originalMessage as IOUMessage)?.paymentType !== CONST.IOU.PAYMENT_TYPE.EXPENSIFY,
```

Comment:
I don't understand yet why we need the cast. [I started a thread](https://github.com/Expensify/App/pull/28570/files#r1477965967). Let me know if you have any insights.

Reply:
@tienifr We got a reply. The `unknown` typing wasn't intentionally introduced. Do you have some ideas on how this typing could be improved?

Reply:
Let's do that then!